src/test/unit/RewardsCoordinatorUnit.t.sol

```diff
@@ -2710,8 +2710,8 @@ contract RewardsCoordinatorUnitTests_processClaim is RewardsCoordinatorUnitTests
         }
 
         // Parse all 3 claim proofs for distributionRoots 0,1,2 respectively
-        IRewardsCoordinator.RewardsMerkleClaim[] memory claims = _parseAllProofs();
-        IRewardsCoordinator.RewardsMerkleClaim memory claim = claims[2];
+        IRewardsCoordinatorTypes.RewardsMerkleClaim[] memory claims = _parseAllProofs();
```

Comment:
nit: inherit `IRewardsCoordinatorTypes` for cleaner syntax

Reply:
Good catch, updated entire file: https://github.com/Layr-Labs/eigenlayer-contracts/pull/911/commits/005d9e7879a3e1b65321203999480e86f9ad9188